<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>509</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>kEIPM</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="RsaSign">
      <attribute name="title">
       <string>公私钥签名</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>导入需要签名的elf文件以及公钥密匙:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="lab_RsaSign_rsa">
               <property name="text">
                <string>公钥:</string>
               </property>
               <property name="buddy">
                <cstring>lineEd_RsaSign_rsa</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEd_RsaSign_rsa"/>
             </item>
             <item>
              <widget class="QPushButton" name="Btn_RsaSign_rsa">
               <property name="text">
                <string>浏览</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="lab_RsaSign_elf">
               <property name="text">
                <string>elf文件:</string>
               </property>
               <property name="buddy">
                <cstring>treeView_RsaSign_elf</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTreeView" name="treeView_RsaSign_elf"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_7">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Btn_RsaSign_Inport">
            <property name="text">
             <string>导入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Btn_Rsa_Sign_Cancel">
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="CASign">
      <attribute name="title">
       <string>证书签名</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_ELF">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>导入需要签名的elf和证书文件:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="lab_CASign_CA">
               <property name="text">
                <string>证书:</string>
               </property>
               <property name="buddy">
                <cstring>lineEd_CASign_CA</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEd_CASign_CA"/>
             </item>
             <item>
              <widget class="QPushButton" name="Btn_CASign_CA">
               <property name="text">
                <string>浏览</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="lab_CASign_elf">
               <property name="text">
                <string>elf文件:</string>
               </property>
               <property name="buddy">
                <cstring>treeView_CASign_elf</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTreeView" name="treeView_CASign_elf">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_6">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>58</width>
              <height>28</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Btn_Inport">
            <property name="text">
             <string>导入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Btn_Cancel">
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Create_Rsa">
      <attribute name="title">
       <string>生成公私钥</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QFrame" name="frame_9">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="lab_GetRsa_inport_privateKey">
            <property name="text">
             <string>生成私钥</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEd_GetRsa_inport_privateKey"/>
          </item>
          <item>
           <widget class="QPushButton" name="Btn_GetRsa_inport_privateKey">
            <property name="text">
             <string>生成</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>导入私钥以生成公钥</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLabel" name="lab_GetRsa_inport_public_privateKey">
            <property name="text">
             <string>导入私钥</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEd_get_inport_public_privateKey"/>
          </item>
          <item>
           <widget class="QPushButton" name="Btn_GetRsa_publicKey">
            <property name="text">
             <string>生成</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>一键生成公私钥:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="lab_GetRsa_privaPath">
               <property name="text">
                <string>私钥地址:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEd_GetRsa_privaPath"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_7" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QLabel" name="lab_GetRsa_publicPath">
               <property name="text">
                <string>公钥地址:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEd_GetRsa_publicPath"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_8" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>245</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="Btn_GetRsa_Create">
               <property name="text">
                <string>一键生成</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Btn_GetRsa_Cancel">
               <property name="text">
                <string>退出</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Create_UserCA">
      <attribute name="title">
       <string>生成用户证书</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lab_User_inputRootCA">
            <property name="text">
             <string>导入根证书:</string>
            </property>
            <property name="buddy">
             <cstring>lineEd_User_InputPath_RootCA</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEd_User_InputPath_RootCA"/>
          </item>
          <item>
           <widget class="QPushButton" name="Btn_User_Visit_RootCA">
            <property name="text">
             <string>浏览</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gBox_UserCA_Inf">
         <property name="title">
          <string>请输入签名信息：</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEd_UserCountry"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lab_UserState">
            <property name="text">
             <string>S 所在省份 (State/Provice)</string>
            </property>
            <property name="buddy">
             <cstring>lineEd_UserState</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEd_UserState"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lab_UserLocality">
            <property name="text">
             <string>L 所在城市 (Locality)</string>
            </property>
            <property name="buddy">
             <cstring>lineEd_UserLocality</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEd_UserLocality"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lab_UserOrganiz">
            <property name="text">
             <string>O 单位名称 (Organization Name)</string>
            </property>
            <property name="buddy">
             <cstring>lineEd_UserOrganization</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEd_UserOrganization"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lab_UserCommon">
            <property name="text">
             <string>CN 公用名称 (Common Name)</string>
            </property>
            <property name="buddy">
             <cstring>lineEd_UserCommon</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEd_UserCommon"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lab_UserCountry">
            <property name="text">
             <string>C 所在国家 (Country)</string>
            </property>
            <property name="buddy">
             <cstring>lineEd_UserCountry</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="lab_Output_UserCA">
            <property name="text">
             <string>生成用户证书地址:</string>
            </property>
            <property name="buddy">
             <cstring>lineEd_OutPutPath_UserCA</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEd_OutPutPath_UserCA"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>246</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>生成</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Create_RootCA">
      <attribute name="title">
       <string>生成根证书</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="gBox_RootCA_Inf">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
         <property name="title">
          <string>请输入签名信息:</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEd_RootCountry"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lab_RootState">
            <property name="text">
             <string>S 所在省份 (State/Provice)</string>
            </property>
            <property name="buddy">
             <cstring>lineEd_UserState</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEd_RootState"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lab_RootLocality">
            <property name="text">
             <string>L 所在城市 (Locality)</string>
            </property>
            <property name="buddy">
             <cstring>lineEd_UserLocality</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEd_RootLocality"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lab_RootOrganiz">
            <property name="text">
             <string>O 单位名称 (Organization Name)</string>
            </property>
            <property name="buddy">
             <cstring>lineEd_UserOrganization</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEd_RootOrganization"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lab_RootCommon">
            <property name="text">
             <string>CN 公用名称 (Common Name)</string>
            </property>
            <property name="buddy">
             <cstring>lineEd_UserCommon</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEd_RootCommon"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lab_RootCountry">
            <property name="text">
             <string>C 所在国家 (Country)</string>
            </property>
            <property name="buddy">
             <cstring>lineEd_UserCountry</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="lab_RootCA">
            <property name="text">
             <string>生成根证书地址:</string>
            </property>
            <property name="buddy">
             <cstring>Line_output_RootCA</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Line_output_RootCA"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Btn_output_RootCA">
            <property name="text">
             <string>生成</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Btn_outClose_RootCA">
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="Manage_model">
        <property name="text">
         <string>管理员模式</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEd_RsaSign_rsa</tabstop>
  <tabstop>Btn_RsaSign_rsa</tabstop>
  <tabstop>treeView_RsaSign_elf</tabstop>
  <tabstop>Btn_RsaSign_Inport</tabstop>
  <tabstop>Btn_Rsa_Sign_Cancel</tabstop>
  <tabstop>lineEd_CASign_CA</tabstop>
  <tabstop>Btn_CASign_CA</tabstop>
  <tabstop>treeView_CASign_elf</tabstop>
  <tabstop>Btn_Inport</tabstop>
  <tabstop>Btn_Cancel</tabstop>
  <tabstop>Manage_model</tabstop>
  <tabstop>lineEd_User_InputPath_RootCA</tabstop>
  <tabstop>Btn_User_Visit_RootCA</tabstop>
  <tabstop>lineEd_UserCountry</tabstop>
  <tabstop>lineEd_UserState</tabstop>
  <tabstop>lineEd_UserLocality</tabstop>
  <tabstop>lineEd_UserOrganization</tabstop>
  <tabstop>lineEd_UserCommon</tabstop>
  <tabstop>lineEd_OutPutPath_UserCA</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>lineEd_RootCountry</tabstop>
  <tabstop>lineEd_RootState</tabstop>
  <tabstop>lineEd_RootLocality</tabstop>
  <tabstop>lineEd_RootOrganization</tabstop>
  <tabstop>lineEd_RootCommon</tabstop>
  <tabstop>Line_output_RootCA</tabstop>
  <tabstop>Btn_output_RootCA</tabstop>
  <tabstop>Btn_outClose_RootCA</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Btn_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
